frontend/src/pages/settings/account/Account.tsx

```diff
@@ -2,28 +2,29 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Box, Grid, Typography, useTheme } from '@mui/material';
 
-import ContentHeader from 'src/components/ContentHeader';
+import { ContentBox, ContentHeader, SettingsSection } from 'src/components';
 import PasswordForm from 'src/features/settings/account/PasswordForm';
 import DeleteAccountForm from 'src/features/settings/account/DeleteAccountForm';
 import EmailAddressForm from 'src/features/settings/account/EmailAddressForm';
 import ExportAllDataForm from 'src/features/settings/account/ExportAllDataForm';
 import SettingsMenu from 'src/features/settings/SettingsMenu';
-import { SettingsSection } from 'src/components';
+
+import { mainSectionBp, settingsMenuBp } from '../layout';
 
 export const AccountPage = () => {
   const { t } = useTranslation();
   const theme = useTheme();
-  // sectionBreakP can be changed independently of breakP
-  const breakP = { xs: 12, sm: 12, md: 9, lg: 9, xl: 10 };
-  const sectionBreakP = breakP;
+
+  // subSectionBreakP can be changed independently of mainSectionBp
+  const subSectionBreakP = mainSectionBp;
```

Comment:
"BreakP" stands for "Breakpoint", doesn't it? If yes, it is wrong to have the "P" capitalized and I would suggest using the whole word for readability.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1142/commits/cda72980366de25a816881ef0832e3c57b0ed346